demisto_sdk/commands/common/logger.py

```diff
@@ -32,9 +32,9 @@ def logging_setup(
     logging.basicConfig()
     l: logging.Logger = logging.getLogger("demisto-sdk")
 
-    levels = {0: logging.INFO, 1: logging.WARNING, 2: logging.DEBUG}
```

Comment:
why change these? Will this affect existing code?

Reply:
this was a bug, the INFO logs didn't work. That's why we see debug logs all the time